src/common/interfaces/api-response.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { MetaPagination } from '../helpers/pagination.helper';
+
+export interface ApiResponse {
+  data: any;
```

Comment:
```suggestion
  data: Object[] | Object;
```

mungkin bisa seperti ini kang pengganti any 😄

Reply:
oh iya, lebih jelas sih ini, siap siap